### PR TITLE
add effect to track Unstable check failure.

### LINKED ISSUE #9637: Feature Request:  add failure effect on unstable action (inventors) flat checks.
a player brought up an interesting issue in that because he is an inventor he doesnt have any way to track if they fail the unstable checks. and if we end a session mid combat we dont have a way of tracking that in the system. i propose that with the inline flat checks to include a failure effect that is just a reminder for the player that they failed their unstable action check.

https://2e.aonprd.com/Traits.aspx?ID=416

### PR DESCRIPTION
**Pull request by ammalagonc**
also, add the Unstable Check to the Explode Action

closes #9637

### CODE REVIEW

File: packs/actions/explode.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
...
+            "value": "<p>You intentionally take your innovation beyond normal safety limits, making it explode and damage nearby creatures without damaging the innovation... hopefully. The explosion deals @Damage[max(2, @actor.level)d6[@actor.flags.pf2e.inventor.explode]|traits:area-damage] damage with a @Check[type:reflex|dc:resolve(@actor.system.proficiencies.classDCs.inventor.value)|basic:true] to all creatures in a @Template[type:emanation|distance:5] around you (if you're wearing or holding the innovation) or around your innovation (if your innovation is a minion).</p>\n<p>At 3rd level, and every level thereafter, increase your explosion's damage by 1d6.</p>\n<p>If you have the breakthrough innovation class feature, you can choose either a 5-foot or @Template[type:emanation|distance:10] for the area when you use Explode; if you have the revolutionary innovation class feature, you can choose a 5-foot, 10-foot, or @Template[type:emanation|distance:15].</p><hr /><p>@Check[flat|dc:15|options:unstable-check|name:PF2E.SpecificRule.Inventor.Unstable.FlatCheck.Label]{PF2E.SpecificRule.Inventor.Unstable.FlatCheck.Label}</p>"
```
Comment (pedrogrullada) [inline packs/actions/explode.json:14]:
```suggestion
            "value": "<p>You intentionally take your innovation beyond normal safety limits, making it explode and damage nearby creatures without damaging the innovation... hopefully. The explosion deals @Damage[max(2, @actor.level)d6[@actor.flags.pf2e.inventor.explode]|traits:area-damage] damage with a @Check[type:reflex|dc:resolve(@actor.system.proficiencies.classDCs.inventor.value)|basic:true] to all creatures in a @Template[type:emanation|distance:5] around you (if you're wearing or holding the innovation) or around your innovation (if your innovation is a minion).</p>\n<p>At 3rd level, and every level thereafter, increase your explosion's damage by 1d6.</p>\n<p>If you have the breakthrough innovation class feature, you can choose either a 5-foot or @Template[type:emanation|distance:10] for the area when you use Explode; if you have the revolutionary innovation class feature, you can choose a 5-foot, 10-foot, or @Template[type:emanation|distance:15].</p>"
```
The unstable check for Explode is already in its damage roll

File: packs/feat-effects/effect-unstable-check-failure.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        },
+        "level": {
+            "value": 0
+        },
+        "publication": {
+            "license": "OGL",
+            "remaster": false,
+            "title": "Pathfinder Guns & Gears"
+        },
+        "rules": [],
```
Comment (pedrogrullada) [inline packs/feat-effects/effect-unstable-check-failure.json:23]: Shouldn't this effect do something? Perhaps have it add to the description of unstable actions with a reminder that they can't perform them while their innovation is malfunctioning.

Comment (ammalagonc) [reply packs/feat-effects/effect-unstable-check-failure.json:23]: Good point.

Let me add that.

Comment (ammalagonc) [reply packs/feat-effects/effect-unstable-check-failure.json]: Check again when you get a moment

Comment (pedrogrullada) [reply packs/feat-effects/effect-unstable-check-failure.json]: Looks ok to me!

**Pull request merged**